Assets/Scripts/Localization/LocalizationLoader.cs

```diff
@@ -33,19 +33,27 @@ void Awake()
 
             //Get the filepath.
             string filePath = Path.Combine(Application.streamingAssetsPath, "Localization");
-            
+
             //Loop through all files.
-            foreach(string file in Directory.GetFiles(filePath))
+            //TODO: Think over the extension ".lang", might change that in the future.
```

Comment:
if everyone does at little clean up to files the modify then it makes it much easy for us to be consistent with our style guide. But dont worry to much about it , no one would turn down your PR due to some mis-formatted comments that you didn't make.